css/style.css

```diff
@@ -780,18 +1935,123 @@
     flex-direction: column;
     padding: 11px 0 55px 48px;
   }
+  
+  .wrap-information,
+  .wrap {
+    height: 772px;
+  }
+
+  .info__title {
+    font-size: 56px;
+    text-align: right;
+    padding: 0 45px 30px 0;
+    margin-top: 20px;
+  }
+
+  .red-first,
+  .details {
+    height: 445px;
+  }
+
+  .description__title,
+  .description__title-most {
+    font-size: 43px;
+    letter-spacing: 3px;
+  }
+
+  .drive,
+  .model-s {
+    font-size: 21px;
+    padding: 25px 25px 64px 0;
+  }
+
+  .icons,
+  .info-order, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)